app/assets/stylesheets/application.css

```diff
@@ -13,7 +13,7 @@
  */
 
 .btn-sul-dlss {
-  background-color: #8c1515;
+  background-color: #00548F;
```

Comment:
can you put this in a [custom property](https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_custom_properties) called `--digital-blue-dark` and then use it here?  That makes it easier to see that it came from the branding guide at https://identity.stanford.edu/design-elements/color/web/ ?